yash-datalake/nested/dataFactory.json

```diff
@@ -84,13 +84,12 @@
             "[parameters('dataFactoryName')]"
           ],
           "properties": {
-            "type": "AzureStorage",
-            "description": "Azure Storage linked service",
+            "annotations": [],

```

Comment:
remove empty properties - see ci results for the complete list

Reply:
Changed as requested